index.html

```diff
@@ -350,66 +350,93 @@ <h3>Use Cases and Requirements</h3>
 specifically:
         </p>
 
-        <ul>
+        <ol>
```

Comment:
Why is this a numbered list?

Reply:
To enable referencing a concept in discussions, etc. subjective decision, but made it easier when we were editing.

Reply:
leaving as <ol> unless you strongly protest.